README.md

```diff
@@ -60,6 +60,11 @@ Flags:
 Global Flags:
   -v, --verbose   verbose output.
 ```
+* There are some ways to set environment variables for a build. If the same key is set in more than one way, the priority is as follows:
+  1. `--env` or `-e` Flag
+  1. `--env-file` Flag
+  1. environment of screwdriver.yaml
+  1. defaultValue (e.g.: `SD_TOKEN`, `SD_API_URL`)
```

Comment:
camel case?
```suggestion
  1. default value (e.g.: `SD_TOKEN`, `SD_API_URL`)
```

Reply:
or did you mean `defaultEnv` ? ( refer to https://github.com/screwdriver-cd/sd-local/pull/74#issue-1033432302 )
https://github.com/screwdriver-cd/sd-local/blob/fa1ece6995a8d836622ec8b92819a06c6ccda06a/launch/launch.go#L140-L146

Reply:
I fixed.